patch to cut bloat which arises from aligning sections to 64k pages.
http://sourceware.org/ml/binutils/2005-05/msg00712.html

2005-06-01  Maciej W. Rozycki  <macro@linux-mips.org>

	* emulparams/elf32btsmip.sh: Unset DATA_ADDR.

Index: ld/emulparams/elf32btsmip.sh
===================================================================
RCS file: /cvs/src/src/ld/emulparams/elf32btsmip.sh,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -p -r1.4 -r1.5
--- ld/emulparams/elf32btsmip.sh	21 Jan 2002 17:19:09 -0000	1.4
+++ ld/emulparams/elf32btsmip.sh	1 Jun 2005 19:26:25 -0000	1.5
@@ -5,5 +5,6 @@
 OUTPUT_FORMAT="elf32-tradbigmips"
 BIG_OUTPUT_FORMAT="elf32-tradbigmips"
 LITTLE_OUTPUT_FORMAT="elf32-tradlittlemips"
+unset DATA_ADDR
 SHLIB_TEXT_START_ADDR=0
 ENTRY=__start
